### PR TITLE
client createAccountAddress should return a Resource\Address

### PR DESCRIPTION
**Pull request by abgit**
fix for issue https://github.com/coinbase/coinbase-php/issues/82

best,
FA

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -295,10 +295,11 @@ public function loadNextAddressTransactions(ResourceCollection $transactions, ar
         $this->loadNextTransactions($transactions, $params);
     }
 
+    /** @return Address */
     public function createAccountAddress(Account $account, Address $address, array $params = [])
     {
         $data = $this->mapper->fromAddress($address);
-        $this->postAndMap($account->getResourcePath().'/addresses', $data + $params, 'toAddress', $address);
+        return $this->postAndMap($account->getResourcePath().'/addresses', $data + $params, 'toAddress', $address);
     }
```
Comment (flowocap) [inline src/Client.php]: c6a49a48e3062af50b2798aa906b44b11c4ea4c4

**Comment by aianus** [comment]
This would be inconsistent with the rest of the library (`createCheckout`, `createOrder`, etc.)

File: src/Client.php (hunk 1 above)
Comment (flowocap) [inline src/Client.php]: Its crazy

Comment (abgit) [inline src/Client.php]: why @flowocap  ?

**Comment by abgit** [comment]
> This would be inconsistent with the rest of the library (createCheckout, createOrder, etc.)

Maybe **all** `create`'s should return the created object.
Anyway, i have already an workaround to make this work.. as expected (at least to me).

**Pull request closed without merging**